### ISSUE TITLE
Spine 4.2 Integration

### ISSUE THREAD

**Issue by Brick2892** [YoYoGames/GameMaker-Feature-Requests#1471]
### Is your feature request related to a problem?

Currently, Spine integration in Game Maker is only up to 4.0 (latest). Since this update, Spine has added many QOL improvements that would make workflow in Spine much easier. It's frustrating to see these improvements and not be able to utilize them.

### Describe the solution you'd like

Full implementation of the latest Spine build which uses physics constraints.

**Comment by KormexGit** [comment]
Yes please, I need the curve editor so bad, and those physics constraints look awesome
